fpm/recipes/ruby-2.4.0/recipe.rb

```diff
@@ -0,0 +1,32 @@
+class RbenvRuby231 < FPM::Cookery::Recipe
+
+  homepage 'https://www.ruby-lang.org/'
+  name 'rbenv-ruby-2.4.0'
+  version '1'
+
+  source "https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.0.tar.gz"
```

Comment:
This string can be single quoted as there are no variables in it. And it'll make rubocop happier.